pkg/module/sync/runner.go

```diff
@@ -275,7 +276,7 @@ func generateModuleStatus(module *common.Module, existStatus *v1beta2.ModuleStat
 		newModuleStatus.Message = module.Template.Err.Error()
```

Comment:
Can we provide unit tests for `generateModuleStatus`? Would make refactoring safer.
Also possible as follow-up I guess.

Reply:
Yes, also makes sense. But again I would like to source this out as this is not the focus of this PR.
https://github.com/kyma-project/lifecycle-manager/issues/2205